pyjs_code_runner/get_cache_dir.py

```diff
@@ -1,10 +1,10 @@
-import appdirs
+import platformdirs
 
 from pathlib import Path
 
 
 def get_appdir():
-    path = Path(appdirs.user_data_dir("pyjs_code_runner", "ThorstenBeier"))
```

Comment:
I removed `ThorstenBeier` here (and `DerThorsten` from `conftest.py` below), as these strings seemed like a remnant of testing code. The `appauthor` parameter should be picked up as whatever the default is automatically?